### PR TITLE
[TS] Regression fixes

### PR DESCRIPTION
**Pull request by dclaux**
## Related Issue
Fixes https://github.com/microsoft/AdaptiveCards/issues/3564
Fixes https://github.com/microsoft/AdaptiveCards/issues/3563

## Description
- Reverted to **stretch** as the default column width.
- Only render the default **"Title"** value for a Fact's title in the designer

These fixes impact Teams and are somewhat urgent. A new npm package should be generated once they are merged.

## How Verified
Verified manually in both adaptivecards-designer-app and in adaptivecards-visualizer

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/3571)

### CODE REVIEW

**Comment by shalinijoshi19** [comment]
are the desktio team consuming the JS renderer off of our npm package or direct from source?

File: source/nodejs/package-lock.json [hunk 1]
```diff
@@ -5757,20 +5754,17 @@
...
-          "dev": true,
-          "optional": true
+          "dev": true
         },
         "console-control-strings": {
           "version": "1.1.0",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```
Comment (shalinijoshi19) [inline source/nodejs/package-lock.json]: Question: Are these intentional changes?

File: source/nodejs/adaptivecards/src/card-elements.ts [hunk 2]
```diff
@@ -5843,9 +5829,9 @@ export class Column extends Container {
         return Enums.Orientation.Vertical;
     }
 
-    width: ColumnWidth = "auto";
+    width: ColumnWidth = "stretch";
```
Comment (shalinijoshi19) [inline source/nodejs/adaptivecards/src/card-elements.ts]: Question: Is this default behavior documented someplace where we will need to update our documentation as well with this change?

**Comment by shalinijoshi19** [comment]
@RebeccaAnne , @paulcam206 , @nesalang could one of you take a look at this PR real quick? SHould be a quick review. tx

File: source/nodejs/adaptivecards/src/card-elements.ts (hunk 2 above)
Comment (RebeccaAnne) [reply source/nodejs/adaptivecards/src/card-elements.ts]: Looks like the default is not doc'd here: [https://adaptivecards.io/explorer/Column.html](https://adaptivecards.io/explorer/Column.html). It probably should be.

Looks like shared model is defaulting to stretch as well.

---
In reply to: [342851467](https://github.com/microsoft/AdaptiveCards/pull/3571#discussion_r342851467) [](ancestors = 342851467)

**Pull request merged**